README.md

```diff
@@ -114,7 +114,7 @@ This command will:
    causing it to download more data in exchange for avoiding processing the entire history
    of the Binance Smart Chain network, which is very CPU intensive.
  * Start up `geth`'s built-in interactive [JavaScript console](https://geth.ethereum.org/docs/interface/javascript-console),
-   (via the trailing `console` subcommand) through which you can interact using [`web3` methods](https://web3js.readthedocs.io/en/) 
+   (via the trailing `console` subcommand) through which you can interact using [`web3` methods](https://web3js.readthedocs.io/en/v1.8.1/) 
```

Comment:
better: https://web3js.readthedocs.io/
which will redirect to the latest version, which is https://web3js.readthedocs.io/en/v1.8.1/ right now.

Reply:
done